### PR TITLE
Fixed polygon and polyline not re-render when changing tile URL

### PR DESCRIPTION
**Pull request by LeMinh1995**

### CODE REVIEW

**Comment by LeMinh1995** [comment]
Hi. Currently, I'm working with UrlTile component. But I found an issue when changing tile URL dynamically. The polygons and polylines are disappeared.

File: lib/ios/AirMaps/AIRMapUrlTile.m [hunk 1]
```diff
@@ -39,6 +39,13 @@ - (void) update
     if (_map == nil) return;
     [_map removeOverlay:self];
     [_map addOverlay:self level:MKOverlayLevelAboveLabels];
+    for (id<MKOverlay> overlay in _map.overlays) {
```
Comment (rborn) [inline lib/ios/AirMaps/AIRMapUrlTile.m]: Won't this have performance issues. Isn't there any way to decide the "zIndex" of an overlay so we don't have to remove and add the other overlays?

Comment (LeMinh1995) [reply lib/ios/AirMaps/AIRMapUrlTile.m]: I have tried insertOverlay with index equal to 0 but still not working.

Comment (LeMinh1995) [reply lib/ios/AirMaps/AIRMapUrlTile.m]: I have tried to use insertOverlay with atIndex equals 0. But still not working. So this is the solution I came up so far.

Comment (rborn) [reply lib/ios/AirMaps/AIRMapUrlTile.m]: I had a look and it seems that it's done the same way for the polylines and the rest of the views. Thank you.

**Comment by rborn** [comment]
LGTM @alvelig 🐽

**Comment by rborn** [comment]
@alvelig 🐽🐽 - aka double 🐽  😹

**Comment by alvelig** [comment]
@rborn thank you for pushing us forward. LGTM. At least I don't have another alternative to come up with.

**Pull request merged**